### PR TITLE
Search: Code formatting and PHPDoc updates.

### PR DESCRIPTION
**Pull request by Viper007Bond**
Just some code formatting cleanup (thanks PhpStorm!) and the addition/fixing of some PHPDocs. No real code changes -- just fixes that PhpStorm was warning me about.

I should have probably saved this for right before code freeze since our code is in flux, but whatever. 🤷‍♂️

### CODE REVIEW

File: modules/search/class.jetpack-search-widget-filters.php [hunk 1]
```diff
@@ -208,30 +222,30 @@ function widget( $args, $instance ) {
...
 				order.val(orderDefault);
 
-				container.find( '.jetpack-search-sort' ).change( function( event ) {
-					var values  = event.target.value.split( '|' );
-					orderBy.val( values[0] );
-					order.val( values[1] );
+				container.find('.jetpack-search-sort').change(function (event) {
+					var values = event.target.value.split('|');
+					orderBy.val(values[0]);
+					order.val(values[1]);
```
Comment (zinigor) [inline modules/search/class.jetpack-search-widget-filters.php]: I can see that in some places your editor removes spaces inside function calls, isn't that against WordPress coding standards?

Comment (Viper007Bond) [reply modules/search/class.jetpack-search-widget-filters.php]: I know that's the case for PHP but I was thinking it was different for JavaScript. Turns out I was wrong:

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#arrays-and-function-calls

I'll fix it. Thanks!

**Comment by ebinnion** [comment]
> thanks PhpStorm!

I do miss that about PhpStorm 😄

**Comment by Viper007Bond** [comment]
This branch has gone to hell. I'm starting over. :)

**Pull request closed without merging**